framework/assets/yii.gridView.js

```diff
@@ -123,7 +123,7 @@
             var $form = $('<form/>', {
                 action: url,
                 method: 'get',
-                class: 'gridview-filter-form',
+                'class': 'gridview-filter-form',
```

Comment:
Quoting single key, while leaving other as they are looks intentional. Quoting all keys looks like chaotic and unnecessary increase file size. 

> Besides, do we realy need to wait until more of those keys become reserved?

I don't think we should worry about it. `class` was reserved in ECMAScrip 1, which has almost 20 years. Reserved words do not appear surprisingly and still can affect variables names, so quoting everything will not solve anything.